OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -201,8 +201,10 @@ public Actor ClosestProc(Actor self, Actor ignore)
 
 			// Start a search from each refinery's delivery location:
 			List<CPos> path;
-			var li = self.Info.TraitInfo<MobileInfo>().LocomotorInfo;
-			using (var search = PathSearch.FromPoints(self.World, li, self, refs.Values.Select(r => r.Location), self.Location, false)
+
+			var locomotor = mobile.Locomotor;
```

Comment:
The old `var` was mostly about reducing line length of the `using`, I think we can drop this now and use `mobile.Locomotor` directly below.